src/python/bot/fuzzers/engine_common.py

```diff
@@ -436,6 +439,23 @@ def get_issue_components(fuzz_target_path):
   return get_issue_metadata(fuzz_target_path, COMPONENTS_FILE_EXTENSION)
 
 
+def get_additional_issue_metadata(fuzz_target_path):
+  """Return the additional metadata fields given a fuzz target path. The data
+  will be a JSON-formatted dictionary."""
+  metadata_file_path = fuzzer_utils.get_supporting_file(
+      fuzz_target_path, METADATA_FILE_EXTENSION)
+
+  if environment.is_trusted_host():
+    metadata_file_path = fuzzer_utils.get_file_from_untrusted_worker(
+        metadata_file_path)
+
+  if not os.path.exists(metadata_file_path):
+    return {}
+
+  with open(metadata_file_path) as handle:
+    return handle.read()
```

Comment:
For consistency with the other return value ({}) in this function on line 453, and other get_* functions, this should return the parsed value (i.e. the dictionary).

Something like this (with extra checking so that bad formats don't propagate all the way into Testcase entities).
 
```python

with open(metadata_file_path) as handle:
  try:
    return json.load(handle)
  except (ValueError, TypeError):
    logs.log_error('Invalid metadata file format.', path=metadata_file_path)
    return {}
```